src/components/ImagePicker/ImagePicker.tsx

```diff
@@ -33,7 +33,7 @@ const ImagePicker = ({ navigation }: any) => {
 
   const widgetSettings = useMemo(
     () => ({
-      getImageMetaData: false, // true might perform slower results but gives meta data and absolute path for ios users
+      getImageMetaData: true, // true might perform slower results but gives meta data and absolute path for ios users
```

Comment:
How do your changes make use of the metadata?

Reply:
Ty for pointing this out -- I forgot to change the value back to false.

I tested this locally and the uploading feature works just fine. I'll edit this file right now.

Reply:
maybe fix in a separate packages PR? 
https://stackoverflow.com/questions/55311228/how-to-remove-warning-async-storage-has-been-extracted-from-react-native-core